zuul.d/whitebox_neutron_tempest_jobs.yaml

```diff
@@ -0,0 +1,129 @@
+          neutron_plugin_options.ipv6_metadata false
+          neutron_plugin_options.advanced_image_ref 11111111-1111-1111-1111-111111111111
+          neutron_plugin_options.advanced_image_flavor_ref 22222222-2222-2222-2222-222222222222
+          whitebox_neutron_plugin_options.openstack_type podified
+          whitebox_neutron_plugin_options.run_traffic_flow_tests True
+          whitebox_neutron_plugin_options.kubeconfig_path '/home/zuul/.crc/machines/crc/kubeconfig'
+          validation.allowed_network_downtime 15
+          validation.run_validation true
+          identity.v3_endpoint_type public
+          identity.v2_admin_endpoint_type public
+      # NOTE(gibi): This is a WA to force the publicURL as otherwise
+      # tempest gets configured with adminURL and that causes test
+      # instability.
+      cifmw_test_operator_tempest_workflow:
+        - stepName: multi-thread-testing
+          tempestRun:
+            concurrency: 6
+            includeList: |
+              whitebox_neutron_tempest_plugin.*
+            excludeList: |
+              neutron_tempest_plugin.scenario.test_mtu.NetworkWritableMtuTest
+              test_multicast.*ext*
+              test_multicast.*restart
+              whitebox_neutron_tempest_plugin.*south_north
+              whitebox_neutron_tempest_plugin.tests.scenario.*test_mtu
+              ^neutron_.*plugin..*scenario.test_.*macvtap
+              ^neutron_tempest_plugin.fwaas.*
+              ^whitebox_neutron_tempest_plugin.*many_vms
+              ^whitebox_neutron_tempest_plugin.*networker_reboot
+              ^whitebox_neutron_tempest_plugin.*ovn_controller_restart
+              ^whitebox_neutron_tempest_plugin.*reboot_node
+              ^whitebox_neutron_tempest_plugin.*test_previously_used_ip
+              ^whitebox_neutron_tempest_plugin.tests.scenario.test_internal_dns.InternalDNSInterruptions.*
+              ^whitebox_neutron_tempest_plugin.tests.scenario.test_l3ha_ovn.*
+              ^whitebox_neutron_tempest_plugin.tests.scenario.test_metadata_rate_limiting
+              ^whitebox_neutron_tempest_plugin.tests.scenario.test_ovn_dbs.OvnDbsMonitoringTest.*
+              ^whitebox_neutron_tempest_plugin.tests.scenario.test_qos.*external
+              ^whitebox_neutron_tempest_plugin.tests.scenario.test_security_group_logging
+              ^whitebox_neutron_tempest_plugin.tests.scenario.test_extra_dhcp_opts.ExtraDhcpOptionsTest.test_extra_dhcp_opts_ipv4_ipv6_stateless
+            # https://review.opendev.org/892839
+            # - neutron_tempest_plugin.scenario.test_mtu.NetworkWritableMtuTest
+            # It's in Blacklist before, FWaaS tests are not executed in any of our setups so there is no need to keep them whitelisted
+            #   ^neutron_tempest_plugin.fwaas.*
+            # Note(chandankumar): Skipping due to https://issues.redhat.com/browse/OSPRH-9091 and It require xl node to fix no valid host.
```

Comment:
Usually with bigger advanced image when we saw 500 no valid host found, is when compute nodes storage wasn't big enough, when we increased disk size, ceph partitions, and computes amount, so each compute would supply at least 120G storage, then we didn't see this error anymore downstream (it also happened in other test cases using vms with advanced image).